eng/pipelines/runtimelab.yml

```diff
@@ -84,151 +84,16 @@ stages:
       buildConfig: Release
       platforms:
       - Linux_x64
```

Comment:
Thanks for fixing it up. I have resolved the first conflict in the file, but forgot to look for more...